src/annotator/delegator.coffee

```diff
@@ -244,7 +244,7 @@ Delegator.natives = do ->
   """
   blur focus focusin focusout load resize scroll unload click dblclick
   mousedown mouseup mousemove mouseover mouseout mouseenter mouseleave
-  change select submit keydown keypress keyup error
+  change select submit keydown keypress keyup error touchstart
```

Comment:
My intention is that in future, to simplify following the logic, we'll remove the `Delegator` infrastructure in future and just use direct event listeners instead (via `addEventListener` on DOM elements or `on<EventName>` in Preact components). For now I've extended what was already there.